src/web/src/components/Banner.tsx

```diff
@@ -148,6 +150,26 @@ export default function Banner() {
     getGitData();
   }, []);
 
+  // Observer banner
+  useEffect(() => {
+    const options = {
+      root: null,
```

Comment:
We need the element to observer here, which should be the entire `<div>` for the banner I think.